Google.Apis.Common/src/Google.Apis.Common/PagingAsyncEnumerable.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright 2015 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Google.Apis.Common
+{
+    /// <summary>
+    /// Implementation of <see cref="IAsyncEnumerable{T}"/> used for pagination.
+    /// </summary>
+    internal sealed class PagingAsyncEnumerable<T> : IAsyncEnumerable<T>
+    {
+        private readonly Func<CancellationToken, Task<IEnumerable<T>>> provider;
```

Comment:
Thanks. I tried running codeformatter on the solution, but it didn't do anything. I wonder whether it doesn't support project.json projects yet :(